src/yb/util/net/dns_resolver.cc

```diff
@@ -136,12 +136,10 @@ Result<InetAddress> PickResolvedAddress(
   if (error) {
     return STATUS_FORMAT(NetworkError, "Resolve failed $0: $1", host, error.message());
   }
-  std::vector<InetAddress> addresses, addresses_v6;
+  std::vector<InetAddress> addresses;
```

Comment:
It was intentional to prefer IPv4 over IPv6 because we already seen users with setup when host resolves to both IPv4 and IPv6 but IPv6 actually was not properly configured.

Reply:
I don't feel like we should be enabling broken setups. If your current IP stack is not working, it should be fixed. Always prefer the current version instead of the legacy version in accordance with IETF Best Current Practices 177 (BCP177). Especially if we now support the current version properly..